src/Employer/Employer.Web/Controllers/ErrorController.cs

```diff
@@ -101,11 +101,10 @@ private IActionResult PageNotFound()
             return View(ViewNames.PageNotFound);
         }
 
-        private async Task<IActionResult> BlockedEmployerAsync()
+        private IActionResult BlockedEmployerAsync()
         {
-            await HttpContext.SignOutEmployerWebAsync();

```

Comment:
The blocked employer story said to explicitly to log out when an employer is blocked.

Reply:
I checked with @mgwilliam  and Faizal and this behaviour is ok. Logging out of just recruit does make much sense and now using the SSO forces a redirect to an Idams logout page.